tests/test_distributedarray.py

```diff
@@ -118,14 +132,22 @@ def test_distributed_math(par1, par2):
 
 
 @pytest.mark.mpi(min_size=2)
-@pytest.mark.parametrize("par1, par2", [(par3_1, par3_2), (par4_1, par4_2)])
+@pytest.mark.parametrize("par1, par2", [(par6, par7), (par8, par9)])
 def test_distributed_dot(par1, par2):
     """Test Distributed Dot product"""
-    pass
+    arr1 = DistributedArray.to_dist(x=par1['x'], partition=par1['partition'], axis=par1['axis'])
+    arr2 = DistributedArray.to_dist(x=par2['x'], partition=par2['partition'], axis=par2['axis'])
+    assert_almost_equal(arr1.dot(arr2), np.dot(par1['x'].flatten(), par2['x'].flatten()), decimal=3)
 
 
 @pytest.mark.mpi(min_size=2)
-@pytest.mark.parametrize("par1, par2", [(par3_1, par3_2), (par4_1, par4_2)])
-def test_distributed_norm(par1, par2):
-    """Test Distributed norm method"""
-    pass
+@pytest.mark.parametrize("par", [(par4), (par4j), (par5), (par5j),
+                                 (par6), (par7), (par8), (par9)])
+def test_distributed_norm(par):
+    """Test Distributed numpy.linalg.norm method"""
+    arr = DistributedArray.to_dist(x=par['x'], axis=par['axis'])
+    assert_array_almost_equal(arr.norm(ord=1, axis=par['axis']),
```

Comment:
Is there any reason that you choose` assert_arrray_almost_equal` for verification instead of `assert_allclose`?
And why you choose `decimal=3`?
I suggest to use latter, and set `rtol=1e-14` for `np.float64` data type. 
Please have a look at https://netlib.org/lapack/lug/node75.html

Reply:
True changed them to `assert_allclose` as it would be better as the dtype of the output will always be a `float` for both complex and float inputs.